src/main.cpp

```diff
@@ -26,7 +26,7 @@ void checkArgs(int port, const std::string& password) {
 
 int main(int ac, char** argv) {
   if (ac != 3) {
-    std::cerr << "Usage: ./ircserv <port> <password>" << std::endl;
+    std::cerr << RED "Usage: ./ircserv <port> <password>" << std::endl;
     exit(EXIT_FAILURE);
   }
 
```

Comment:
In general, a "usage message" is not considered an error. It is typically displayed to inform the user of the correct way to run the program. It would be better to print it in white to distinguish it from actual errors.

Reply:
ok, jai enleve aussi le RED pour le cerr de check args (invalid port or password too long).